src/js/content_scripts/input-mapper-base.js

```diff
@@ -288,19 +304,18 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
         var inputProperties = that.model.map[inputType][index],
             actionLabel = fluid.get(inputProperties, "currentAction") || fluid.get(inputProperties, "defaultAction");
 
+        /**
+         * TODO: Modify the action call in such a manner that the action gets triggered
+         * when the inputs are released. (To gain shortpress and longpress actions)
```

Comment:
Once you've got the core functionality on board in some form (performing an action using at least one type of input), we should review and refactor and make each "style" of interaction into a clearer pattern and ensure that all functions can be used with:

1. Short presses that trigger an action on button release.
2. Long presses that trigger an action on button release after a configured time period.
3. Long presses that repeatedly trigger an action.

You'll need a kind of matrix for these, where you support each type of action for digital buttons, analogue buttons, and axes.  This is probably one to handle later, for example, after you've got the configuration panel working.